components/HomePage/Timeline.js

```diff
@@ -168,7 +168,7 @@ const TimelineSection = ({ event, className }) => {
       };
     })
     .sortBy(m => {
-      return moment(m);
+      return moment(m.due);
```

Comment:
I thought the field is `dueDate`, not `due`.

Reply:
@brettski there is as a `.map` chained prior to this.